tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -234,44 +299,86 @@
    "outputs": [],
    "source": [
     "if credential_manager:\n",
+    "    # OLIApi is used as a context manager\n",
     "    with OLIApi(credential_manager) as oliapi:\n",
     "        # create a new DBS file\n",
+    "        # alternative thermo_frameworks and databanks are available.\n",
+    "        \n",
+    "        # use keep_file = True to save a DBS file ID on the Cloud\n",
+    "        # for use in more than one session\n",
     "        dbs_file_id = oliapi.get_dbs_file_id(\n",
-    "            chemistry_source=source_water[\"components\"],\n",
+    "            inflows=inflows,\n",
+    "            thermo_framework=\"MSE (H3O+ ion)\",\n",
     "            phases=[\"liquid1\", \"solid\"],\n",
+    "            model_name=\"test\",\n",
+    "            keep_file=true,\n",
```

Comment:
```suggestion
    "            keep_file=True,\n",
```

Reply:
As is, this also produces an error. @lbianchi-lbl I definitely think it is important to incorporate testing for this notebook in our GH checks at some point, since things can break easily if API changes are not also reflected here.

Reply:
@adam-a-a I've added notebook testing to the OLI API GHA check suite, see https://github.com/watertap-org/watertap/issues/1298#issuecomment-1969208395.